05-mvc-with-rails/error-handling/readme.md

```diff
@@ -0,0 +1,16 @@
+# Error Handling
+
+## Learning Objectives
+
+- Explain the benefits of explicitly handling errors
+- Produce and handle an error in Javascript using the keywords `try`, `catch`, `throw`, and `finally`
```

Comment:
I'd propose removing this and making a note to cover this in week 7. Just because there's a lot of new terminology and concepts in this lesson, and the more we can trim, I think, the better.